source/future.tex

```diff
@@ -51,6 +51,15 @@
 \rSec1[depr.c.headers]{C standard library headers}
 
 \pnum
+For compatibility with the C standard library\indextext{library!C standard},
+the \Cpp standard library provides headers
+\tcode{<ccomplex>}~(\ref{depr.ccomplex.syn}),
```

Comment:
@tkoeppe: The entire clause talks about compatibility features.
Maybe just say "Headers"? And &lt;ccomplex> and friends aren't really for compatibility with C, despite your edit saying so.